data/planners/788107d5-dc1e-4204-9269-38df0186d3e7.yml

```diff
@@ -2,5 +2,12 @@
 
 id: 788107d5-dc1e-4204-9269-38df0186d3e7
 name: sequential
+description: |
+  CALDERA ships with a default planner, sequential. During each phase of the operation,
+  the sequential planner loops through all agents (which are part of the operation's group)
+  and sends each of them a list of all ability commands the planner thinks it can complete.
+  This decision is based on the agent matching the operating system (execution platform) of
+  the ability and the ability command having no unsatisfied variables. It then waits for each
+  agent to complete their list of commands before moving on to the next phase.
```

Comment:
We've added in "phase-less" operations as well, which will dump all possible actions (based on available facts and so on) of the entire adversary into a single phase which will then run to completion and finish the operation.  Let's add a small note to address that, then it's ready for merge.